### PR TITLE
Fix 3973 fix negative number

### PR DESCRIPTION
**Pull request by ismailsunni**
### What does it fix ?
* Ticket #3973 
* Funded by : DFAT
* Description : 
   All exposure for volcanic ash are enabled now.
screen shot 2017-03-01 at 07 54 54

### CODE REVIEW

File: safe/gis/vector/from_counts_to_ratios.py [hunk 1]
```diff
@@ -69,19 +76,17 @@ def from_counts_to_ratios(layer, callback=None):
                     count_field=count_field['key']))
 
     if len(mapping) == 0:
-        # There is not a count field. Let's skip this layer.
+        # There is not a subset count field. Let's skip this layer.
         layer.commitChanges()
         return layer
```
Comment (Gustry) [inline safe/gis/vector/from_counts_to_ratios.py]: Can you add an information debug message about what we are doing here, with the textual formula?

**Comment by Gustry** [comment]
Great thanks. I put one small comment.
So we will have to revamp the size when we will use feature value? To divide with the size, right ?

**Comment by Gustry** [comment]
I guess you are not working on ticket #3972 as mentioned in this PR. It's #3984

**Comment by felix-yew** [comment]
Hi mas @ismailsunni I try your PR for #3973 
image
:+1: LGTM, thank you

**Comment by samnawi** [comment]
Yeah.... It's work 
image

It's also fix the wrong calculation in the count ratio @ismailsunni ?

File: safe/gis/vector/from_counts_to_ratios.py (hunk 1 above)
Comment (ismailsunni) [inline safe/gis/vector/from_counts_to_ratios.py]: Done.

**Comment by ismailsunni** [comment]
> So we will have to revamp the size when we will use feature value? To divide with the size, right ?

Yes. But I think you have already done that. I check in the `test_ratios_with_vector_exposure`. Divided exposure polygon has the count proportional to the size / area. Not sure about feature value, though.

> I guess you are not working on ticket #3972 as mentioned in this PR. It's #3984

ah yes, typos :D 

Thanks @samnawi @felix-yew I will merge this after Travis is passing.

**Pull request merged**